pages/posts/2017/12-december/140-continuous-integration-at-subvisual.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 140
+path: /posts/140-continuous-integration-at-subvisual/
+title: "Continuous Integration at Subvisual"
+author: miguel-palhas
+date: 14/12/2017
+tags:
+  - development
+intro: "Learn how we do Continuous Integration, and what tools we use at each step of the way"
+---
+
+We're often approached by other companies or developers who want to know more about how we handle our projects' testing and deployment.
+
+Especially since we build and ship multiple times a day and often have multiple developers working in multiple branches simultaneously.
```

Comment:
I'm probably going to leave some comments about sentences that I think would be better written in a different way, but I'm not saying that there is something wrong with them, and maybe the suggestion will obfuscate your own style, I'm just saying that I would write them in another way, if there is something I think isn't correct I'll leave a comment with that. This one could be something like this: `They want to know how we ship multiple times a day while having multiple developers working at the same time.`